02/5_extern_keyword/external_file.c

```diff
@@ -0,0 +1,10 @@
+#include "external_file.h"
+
+int g_extern_var = 0;

```

Comment:
bad naming

Reply:
it is jus a global variable..

also try to use real world like examples.. 

this is not good code examples at all